### DISCUSSION TITLE
Increase Length of Embedded Docs

### DISCUSSION THREAD

**Discussion by edwarddarrah** [outline/outline#4603] [Ideas]
Suggestion to have all embedded files (e.g., Google Docs, Sheets, Slides, etc..) the length of a standard page. Or have a toggle to lengthen the embedded file.

![Outline Screenshot](https://user-images.githubusercontent.com/96745845/209294671-96d7f8f7-6b42-495b-9a8d-d858e6f279c7.png)

**Comment by tommoor** [comment]
Yea, it's not really designed to have an embed just replace a document entirely - we need some more flexibility with display options but it's blocked by changes to the underlying data structure

**Comment by edwarddarrah** [comment]
@tommoor Sorry I never replied to this. Is there currently any consideration to make the files more expandable? Or is it too much of an undertaking with the current structure?
